ironic.conf

```diff
@@ -17,6 +17,8 @@ enabled_raid_interfaces = no-raid,irmc,agent,fake
 enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake
 rpc_transport = json-rpc
 use_stderr = true
+# NOTE(dtantsur): the default md5 is not compatible with FIPS mode
+hash_ring_algorithm = sha256
```

Comment:
Does this influence hashes which end up in the DB?  I'm just wondering of the upgrade implications if the ironic containers get restarted but not mariadb?

In practice on a major upgrade I guess they will all restart, but I wanted to be sure what the impact would be, e.g do we need to verify/enforce that to purge the ephemeral DB?

Reply:
Nope, hashes only exist in memory (generated from host names in the DB, which are dynamic).

Reply:
Ok thanks for clarifying!